lms/authentication/_helpers.py

```diff
@@ -26,15 +26,15 @@ def authenticated_userid(lti_user):
 
 def get_lti_user(request):
     """
-    Return an :class:`~lms.values.LTIUser` for the authenticated LTI user.
+    Return an models.LTIUser for the authenticated LTI user.
```

Comment:
I'm removing these Sphinx `:class:` references and other similar things. They aren't checked or used by anything (e.g. we don't use any tooling that parses them) and it's hard to get people to write them consistently. The syntax is hard to remember.

Reply:
Yeah, it's a pain and we don't make any use of it.